app/src/main/java/fr/iut_rodez/pathpilot_android_client/home/clients/ClientService.java

```diff
@@ -26,9 +27,11 @@
 import fr.iut_rodez.pathpilot_android_client.home.Home;
 import fr.iut_rodez.pathpilot_android_client.model.Client;
 import fr.iut_rodez.pathpilot_android_client.model.Client.ClientArrayAdapter;
+import fr.iut_rodez.pathpilot_android_client.util.network.NetworkUtils;
 
 /**
  * Service to handle all client related requests
+ *
  * @author François de Saint Palais
```

Comment:
No need to put author

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/53/commits/d9bb0f4ac19fc0b609e1f6a9e6b4557779a0ee49